dnf-behave-tests/launch-test

```diff
@@ -32,6 +32,14 @@ if [ "$DNFCOMMAND" ]; then
     DEFINES="$DEFINES -D dnf_command=$DNFCOMMAND"
 fi
 
+if [ "$RESERVE" == "-r" ]; then
```

Comment:
It looks fine, but why to not preserve all directory when tests run in container?

Reply:
I've described that in the commit message :wink: 

To extend on that, try to run the tests with the `preserve` flag, you'll see it prints additional text for each scenario telling you about the directories it didn't delete. This is because if you don't run in a container and run the test suite multiple times, your /tmp gets filled up pretty bad.

But in regular containerized runs it's better to not add noise to the output.